frontend/src/pages/LogsPage/LogsHistogram/LogsHistogram.tsx

```diff
@@ -219,6 +227,15 @@ const LogsHistogram = ({
 		)
 	}
 
+	let loadingState: LoadingState | undefined
+	if (showLoadingState) {
+		if (outline) {
+			loadingState = 'spinner'
+		} else {
+			loadingState = 'skeleton'
+		}
+	}
```

Comment:
Got feedback that it was hard to tell that there was work going on with the skeleton view. I think on the logs page it is more clear because there's also a loading spinner in place of the log count, but we don't show that on the log alerts page.

Reply:
Got it - makes sense. Thanks for clarifying!